include/VecCore/Backend/Implementation.h

```diff
@@ -284,6 +285,21 @@ VECCORE_FORCE_INLINE VECCORE_ATT_HOST_DEVICE constexpr Bool_s EarlyReturnAllowed
 #endif
 }
 
+// An extension of EarlyReturnAllowed(), and new name is a reminder that it can be used for other purposes as well
+template <typename T, size_t simdSize = 1>
+VECCORE_FORCE_INLINE
+VECCORE_ATT_HOST_DEVICE
+constexpr Bool_s SIMDsizeUpTo()
```

Comment:
I'm starting to think that it's just better to keep this in VecGeom. The name `ERVecSizeAtMost` is not very clear, as ER is abbreviated and not obvious, while the longer name does not save much typing at all, which is the intention here. I know you think `EarlyReturnAllowed() && VectorSize<T>() <= N` is too much to type, but I think it's much clearer. I don't see an advantage in providing a function with a cryptic name just to save a few characters on if statements in VecGeom, as that involves extra testing code as well for something that is already possible to do with a few more characters.